pyroSAR/drivers.py

```diff
@@ -1500,29 +1512,40 @@ def unpack(self, directory, overwrite=False):
 
 class Archive(object):
     """
-    Utility for storing SAR image metadata in a spatialite database
+    Utility for storing SAR image metadata in a database
 
     Parameters
     ----------
     dbfile: str
-        the database file. This file might either point to an existing database or will be created otherwise.
+        the database name. This might either point to an existing database or will be created otherwise.
```

Comment:
Please add a clarifying comment that `dbfile` will be a file name if creating a SQLite database.